agent/engine/docker_image_manager_integ_test.go

```diff
@@ -37,6 +39,13 @@ const (
 	testImage3Name = "127.0.0.1:51670/amazon/image-cleanup-test-image3:latest"
 )
 
+const credentialsIdInteTest = "credsid"
+
+func defaultTestConfigInteTest() *config.Config {
```

Comment:
This looks like a spelling error; can you fix it to `defaultTestConfigIntegTest`?